modules/Emails/Email.php

```diff
@@ -1929,6 +1929,30 @@ public function getForwardHeader()
         //return from_html($ret);
     }
 
+    public function getEmailHeaderOutgoing($emailCc)
+    {
+        global $mod_strings;
+        global $current_user;
+
+        $mailAccount = BeanFactory::getBean('InboundEmail', $this->mailbox_id);
+
+        $toEmail = $mailAccount->email_user;
+
+        //$from = str_replace(array("&gt;","&lt;"), array(")","("), $this->from_name);
```

Comment:
Hi @cameronblaikie, thank you for your PR, can you remove the commented code please?